packages/rating-scales/src/discrete/Radio.jsx

```diff
@@ -60,10 +62,10 @@ const SecondaryRadioLabel = ({ style, ...p }) => (
 /**
  * A styled radio button input
  */
-const RadioInput = ({ style, ...p }) => (
+const RadioInput = ({ name, style, ...p }) => (
   <input
     type="radio"
-    name="discrete"
+    name={name || "descrete"}
```

Comment:
Just a check that you don't want this to be `discrete`? (given directory and other code I think)

Reply:
fixed